tests/scenarios/protocol-message-size-limits.spec.ts

```diff
@@ -0,0 +1,109 @@
+import type { EventStream } from '../../src/types/subscriptions.js';
```

Comment:
Great tests! 🥇 

We probably don't need a scenario section fort this test as it isn't huge.
Might fit well around here since it relates to protocol write rules specifically:

https://github.com/TBD54566975/dwn-sdk-js/blob/b4e09343946d231df19a86d9aed79619fec5992b/tests/handlers/records-write.spec.ts#L1166